### PR TITLE
docs: view ecosystem partners

### PR DESCRIPTION
**Pull request by angelicawill**
resolves https://github.com/trufnetwork/truf-network/issues/117

### CODE REVIEW

**Review by MicBun — CHANGES_REQUESTED** [review]
image

Frax Finance is not rendered here @angelicawill

**Comment by angelicawill** [comment]
@micbun what do you use to check? it's rendered in my local 
CleanShot 2024-12-19 at 10 13 01@2x

File: ecosystem-partners.mdx [hunk 1]
```diff
@@ -0,0 +1,562 @@
...
+<div className="not-prose grid gap-x-4 sm:grid-cols-4">
+	<PartnerCard
+		title="Ampleforth"
+		href="https://www.ampleforth.org/"
+		img="images/partners/ampleforth.png"
+	></PartnerCard>
+	<PartnerCard
+		title="FRAX"
+		href="https://frax.finance"
+		img="images/partners/frax.png"
```
Comment (MicBun) [inline ecosystem-partners.mdx:455]:
```suggestion
		img="images/partners/Frax.png"
```

or just rename Frax.png into frax.png

**Comment by MicBun** [comment]
> @MicBun what do you use to check? it's rendered in my local

@angelicawill, It seems our system handles case-sensitive file names differently.

**Review by MicBun — APPROVED** [review]

**Comment by MicBun** [comment]
nice

**Pull request merged**